pubsub.go

```diff
@@ -43,6 +46,19 @@ type ReceiveSettings struct {
 	Settings pubsub.ReceiveSettings
 }
 
+// SubscriptionConfigs is an extension of Google PubSub's SubscriptionConfig that
+// enables further configuration for a subscription of a topic
+type SubscriptionConfigs struct {
```

Comment:
How would you feel about making the Config part singular so `SubscriptionConfig` vs `SubscriptionConfigs`?

Also what are your thoughts around baking in the other parameters from `CreateSubscriptions` into the config struct? So topic id and subscription ids would be within the `SubscriptionConfig` struct and `CreateSubscriptions` would just take the one config object as the parameter?